### PR TITLE
MNT: Add names to Zenodo

### PR DESCRIPTION
**Pull request by effigies**
@kfinc @rciric @mgxd I'm adding you to the .zenodo.json file. Could you update the spelling of your names to how you'd want them listed on a publication (Last, First), and update your affiliation and ORCID? You can use the suggestion feature or just post them in this thread.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -20,6 +20,21 @@
       "name": "Halchenko, Yaroslav O.",
       "orcid": "0000-0003-3456-2493"
     },
+    {
+      "affiliation": "",
```
Comment (kfinc) [inline .zenodo.json]: Centre for Modern Interdisciplinary Technologies, Nicolaus Copernicus University in Toruń

File: .zenodo.json [hunk 2]
```diff
@@ -20,6 +20,21 @@
...
+      "name": "Finc, Karolina",
+      "orcid": ""
```
Comment (kfinc) [inline .zenodo.json]: 0000-0002-0157-030X

File: .zenodo.json [hunk 3]
```diff
@@ -20,6 +20,21 @@
...
+    },
+    {
+      "affiliation": "",
+      "name": "Ciric, Rastko",
+      "orcid": ""
+    },
+    {
+      "affiliation": "",
+      "name": "Goncalves, Mathias",
+      "orcid": ""
```
Comment (mgxd) [inline .zenodo.json]:
```suggestion
      "orcid": "0000-0002-7252-7771"
```

File: .zenodo.json [hunk 4]
```diff
@@ -20,6 +20,21 @@
...
+      "name": "Finc, Karolina",
+      "orcid": ""
+    },
+    {
+      "affiliation": "",
+      "name": "Ciric, Rastko",
+      "orcid": ""
+    },
+    {
+      "affiliation": "",
```
Comment (mgxd) [inline .zenodo.json]:
```suggestion
      "affiliation": "MIT",
```

**Comment by effigies** [comment]
@rciric Can you verify this is how you want to be cited?

**Pull request merged**